File: C#/WPF/CarShop/CarClassLibrary/Car.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace CarClassLibrary
{
    public class Car
    {
        //Declare variables with their Getters and Setters
        public string Make { get; set; }
        public string Model { get; set; }
        public string Color { get; set; }
        public int Year { get; set; }
        public decimal Price { get; set; }
        

        //Constructors
        public Car (string make, string model, string color, int year, decimal price)
        {
            Make = make;
            Model = model;
            Color = color;
            Year = year;
            Price = price;
        }

        public Car ()
        {
            Make = "Nothing yet";
            Model = "Nothing yet";
            Color = "Nothing yet";
            Year = 0;
            Price = 0;
        }

        public string Display
        {
            get
            {
                return string.Format("{0} {1} {2} {3} ${4}", Year, Color, Make, Model, Price);
            }
        }
    }
}

File: Java/Smaller Projects/InterceptorPattern/src/bean/Car.java
package bean;
import javax.ejb.Stateless;
import javax.interceptor.Interceptors;
import interceptor.Interceptor;

@Stateless
@Interceptors(Interceptor.class)
public class Car 
{
	
	//Variables
	private String make;
	private String model;
	private String color;
	private int year;
	
	//Constructors
	@Interceptors(Interceptor.class)
	public Car()
	{
		
	}
	public  Car(String make, String model, String color, int year)
	{
		this.make = make;
		this.model = model;
		this.color = color;
		this.year = year;
	}
	
	//Getters and Setters
	public String getMake()
	{
		return make;
	}
	public void setMake(String make)
	{
		this.make = make;
	}
	public String getModel()
	{
		return model;
	}
	public void setModel(String model)
	{
		this.model = model;
	}
	public String getColor()
	{
		return color;
	}
	public void setColor(String color)
	{
		this.color = color;
	}
	public int getYear()
	{
		return year;
	}
	public void setYear(int year)
	{
		this.year = year;
	}
}
